### ISSUE TITLE
Rename python to python2 and add separate Python 3.x implementation

### ISSUE THREAD

**Issue by Slko** [bvanrijn/open-fibonacci#2]
Python 2 is slowly dying and will lose support in the future. Python 3 has become the default Python interpreter in some Linux distributions, so having incompatible implementation as a default one (because it is named just "python") can be confusing for some people. It is better to use either one single implementation that supports both versions, or two separate implementations named python2 and python3.

**Comment by bvanrijn** [comment]
Good point. I'll add a version that is compatible with both Python 2.7 and 3.5 tomorrow, or better yet split them up.
